controllers/digital-fund/strand-2.js

```diff
@@ -10,11 +10,6 @@ const features = config.get('features');
 
 const router = express.Router();
 
-router.use((req, res, next) => {
-    res.locals.pageAccent = 'cyan';
```

Comment:
These are going because `pageAccent` doesn't make sense any more in the new brand. But we will want to come back to this as and when we relaunch the digital fund. I know @LynseyJReynolds is keen to keep _some_ kind of distinction between the two strands.